### PR TITLE
travis enhancement

### PR DESCRIPTION
**Pull request by DravitLochan**
Fixes issue #47 

Changes: travis was building development branch which does not exists. Updated travis for master branch

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,38 @@
...
+
+before_install:
+  - export CHROME_BIN=chromium-browser
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+before_script:
+  - npm install -g karma
+  - npm install
+  - npm build package.json
```
Comment (betterclever) [inline .travis.yml]: it should be ```npm run build```

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,38 @@
...
+  - sh -e /etc/init.d/xvfb start
+
+before_script:
+  - npm install -g karma
+  - npm install
+  - npm run build
 
 script:
   - npm test
+  - karma start config/karma.conf.js --single-run
```
Comment (betterclever) [inline .travis.yml]: here you are invoking karma script with config file in config/karma.conf.js, the file is not present there in the current state of repository. Please add those files as well in the PR or no do test with karma for now.

**Comment by DravitLochan** [comment]
@betterclever @rishiraj824 Removed karma test. Please review!

**Comment by DravitLochan** [comment]
@mariobehling merge this please!

**Pull request merged**